CHANGELOG.md

```diff
@@ -49,6 +49,7 @@ functionality, under the "Removed" section.
   - `nh search` displays a link to the `package.nix` file on the nixpkgs GitHub,
     and also fixes the existing links so that they no longer brokenly point to a
     non-existent file path on nix flake systems.
+- A new `nh os build-image` subcommand has been added to allow building specific nixos images or just a nixos image. See `nh os build-image --help` to see more details on this subcommand. 
```

Comment:
How many times do you plan to misspell "NixOS"? I do not wish to be antagonizing but this displays a lack of attention to your own PR.

Also, this changelog entry is too vague. It should explain what image variants (which is, for some reason, freeform) are supported and provide an example command.